examples/simple/src/Simple.es6

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import Valya from '../../../build/index';
```

Comment:
I'm not sure about how to test this example live, maybe we need to create separate repo with webpack and other env stuff?